### ISSUE TITLE
Will you add parallel python into keras' load_data module because of python just use one core

### ISSUE THREAD

**Issue by meanmee** [fchollet/keras#1009]

**Comment by EderSantana** [comment]
users can define custom datasets, they just have to complain to a regular numpy API. See the HDF5Tensor for instance. https://github.com/fchollet/keras/blob/master/keras/utils/io_utils.py#L7-L71 Inside the slice method, you could do throw a thread and do data processing in parallel.

About supporting that in Keras natively, I haven't heard people discussing that on "future work" threads yet...

**Comment by meanmee** [comment]
yes，real-time data augmentation needs parallel。 发自网易邮箱大师 On 2015-11-16 12:01 , Eder Santana Wrote: users can define custom datasets, they just have to complain to a regular numpy API. See the HDF5Tensor for instance. https://github.com/fchollet/keras/blob/master/keras/utils/io_utils.py#L7-L71 Inside the slice method, you could do throw a thread and do data processing in parallel. About supporting that in Keras natively, I haven't heard people discussing that on "future work" threads yet... — Reply to this email directly or view it on GitHub.
